actions/admin/settings/160.nameserver.php

```diff
@@ -115,6 +115,28 @@
 					'save_method' => 'storeSettingField',
 					'required_otp' => true
 				],
+				'system_bindmasterservers' => [
+					'label' => lng('serversettings.bindmasterservers'),
+					'settinggroup' => 'system',
+					'varname' => 'bindmasterservers',
+					'type' => 'text',
+					'string_type' => 'validate_ip_incl_private',
+					'string_delimiter' => ',',
+					'string_emptyallowed' => true,
+					'default' => '',
+					'save_method' => 'storeSettingField',
+					'visible' => Settings::Get('system.dns_server') == 'Bind'
+				],
+				'system_bindmirrorconf_directory' => [
```

Comment:
This is only used for one single/fixed file, maybe we could make it a complete setting including the filename (`string_type => file`) and give more control to the admin instead of having a fixed filename?